library/Zend/Session/Service/SessionManagerFactory.php

```diff
@@ -113,16 +112,14 @@ public function createService(ServiceLocatorInterface $services)
             ) {
                 $managerConfig = array_merge($managerConfig, $configService['session_manager']);
             }
-            // Attach validators to session manager, if any
```

Comment:
The old code attached validators at that point (`$chain->attach(...)`). That code has been removed and there are no validators attached at that point anymore, which makes the comment obsolete.